packages/test/src/test-webpack.ts

```diff
@@ -0,0 +1,27 @@
+import { Worker } from '@temporalio/worker';
```

Comment:
I think `test-bundler.ts` might be more appropriate.

Also considering that it might be better for future maintainability if we merge this with `test-worker-from-bundle.ts`... That's where most bundler-related tests are at the moment, including some tests that are not related to workers.

Reply:
good idea, thanks